crates/re_data_ui/src/item_ui.rs

```diff
@@ -668,9 +668,11 @@ pub fn entity_db_button_ui(
     }
 
     if response.clicked() {
-        // Open the recording / switch to this blueprint…
-        ctx.command_sender
-            .send_system(SystemCommand::ActivateStore(store_id.clone()));
+        // Open the recording
+        if store_id.kind == re_log_types::StoreKind::Recording {
+            ctx.command_sender
+                .send_system(SystemCommand::ActivateRecording(store_id.clone()));
+        }
```

Comment:
so clicking a blueprint no longer activates it. I guess that makes sense, but we should probably leave a comment here explaining why!